### ISSUE TITLE
Simple game-related tips if the user does something wrong (e.g. tries to put a card in the wrong place)

### ISSUE THREAD

**Issue by jan-kleks** [KyleU/solitaire.gg#44]
So for instance if the user tries to put a card in the wrong place a message (text or some small window?) would show up giving the user a simple gameplay-related tip and disappear when the users starts to play correctly. A tip could be, for instance, "Build down regardless suit (ex. [...])", including these nice card symbols (check the bottom of the screenshot):

![image](https://cloud.githubusercontent.com/assets/15075571/25463465/c8f9b0fa-2af6-11e7-986f-bfef46394f8d.png)

As you can see most solitaire apps use status bars for such tips, but maybe you can do without them.

**Comment by jan-kleks** [comment]
Of course, the "solitaire teacher" demo mode, [I mentioned here](https://github.com/KyleU/solitaire.gg/issues/16#issuecomment-293403839) might work well too.

**Comment by KyleU** [comment]
I can't do specific advice about the source and the target ("You tried to drop an Five on a Nine, which only accepts Eight"). When you drop a card, I check the mouse pointer and each corner for overlapping decks, meaning every card drop could be overlapping multiple piles, so I never know for sure what move you intended. I could say ("You tried to drop an Five, which only fits on a six"), but I'm not sure how much value that would provide.

**Comment by jan-kleks** [comment]
But this tip in the picture is pretty general too. I guess it might work...
